drivers/display/ssd1306.py

```diff
@@ -14,6 +14,7 @@
 SET_PAGE_ADDR = const(0x22)
 SET_DISP_START_LINE = const(0x40)
 SET_SEG_REMAP = const(0xA0)
+SET_IREF_SELECT = const(0xAD)
```

Comment:
should this go after `SET_MUX_RATIO` so it's sorted by register (hex) value?

Reply:
🤦